std/file.d

```diff
@@ -4178,8 +4178,8 @@ auto dirEntries(string path, string pattern, SpanMode mode,
  * Reads a file line by line and parses the line into a single value or a
  * $(REF Tuple, std,typecons) of values depending on the length of `Types`.
  * The lines are parsed using the specified format string. The format string is
- * passed to $(REF formattedRead, std,format), and therefore must conform to the
- * format string specification outlined in $(MREF format).
+ * passed to $(REF formattedRead, std,_format), and therefore must conform to the
+ * format string specification outlined in $(MREF _format).
```

Comment:
It points to `format_.html` (underscore is on the other side). Should be: `$(MREF std,_format)`.